slides/03_workflow.qmd

```diff
@@ -245,15 +245,16 @@ Idea: put the code into a package
 1.  Describe the purpose and scope of the package
 2.  Analyse and describe the requirements in clear and simple terms ("prose")
 
-| Obligation level | Key word[^2] | Description    |
-|------------------|:-------------|----------------|
-| Duty             | shall        | "must have"    |
-| Desire           | should       | "nice to have" |
-| Intention        | will         | "optional"     |
+| Obligation level | Key word[^2]     | Description    |
+|------------------|:-----------------|----------------|
+| Duty             | must, shall      | "must have"    |
+| Desire           | should           | "nice to have" |
+| Intention        | may              | "optional"     |
```

Comment:
I've changed this because to me "will" and "shall" both suggest "must have". In legal writing, "must" may be preferable to "shall" to indicate an obligation, so I have added this as well. I also changed the requirements paragraph on the subsequent slide.